### PR TITLE
Respect smtp port when sending email

### LINKED ISSUE #87: Unable to send emails when using ProtonMail bridge
([protonmail bridge](https://github.com/ProtonMail/proton-bridge) runs a _local_ IMAP server)

Listing and reading emails work, but for some reason I cannot send any emails using himalaya.

```
❯ h -l trace  write
Logger initialized
Output format: PLAIN
Log level: TRACE
Subcommand matched: write
(s)end, (d)raft, (e)dit or (q)uit? s
Sending message…
Connection error: Could not connect
Connection error: Could not connect
Connection error: Could not connect
Connection error: Could not connect
Connection error: Could not connect
Connection error: Could not connect
Connection error: Could not connect
internal client error: timed out waiting for connection: Connection error: Could not connect
```

There is nothing in the protonmail-bridge log, so I suspect the request is not even reaching it.

Other email clients work. 

Let me know if there is a way to provide more diagnostic info.

### PR DESCRIPTION
**Pull request by srid**
Fixes #87

### CODE REVIEW

File: src/smtp.rs [hunk 1]
```diff
@@ -21,7 +21,10 @@ pub fn send(account: &Account, msg: &lettre::Message) -> Result<()> {
         SmtpTransport::relay
     };
 
-    smtp_relay(&account.smtp_host)?
+    let builder = smtp_relay(&account.smtp_host)?;
```
Comment (soywod) [inline src/smtp.rs:24]: Is this intermediate variable necessary?

File: src/smtp.rs [hunk 2]
```diff
@@ -21,7 +21,10 @@ pub fn send(account: &Account, msg: &lettre::Message) -> Result<()> {
...
+
+    builder
+        .port(account.smtp_port)
         .credentials(account.smtp_creds()?)
         .timeout(Some(Duration::new(1000, 0)))
```
Comment (soywod) [inline src/smtp.rs:29]: Can you also remove this line since it was to debug the timedout issue?

**Review by soywod — CHANGES_REQUESTED** [review]

**Comment by soywod** [comment]
Thanks a lot for all your contributions :heart:

**Comment by AckslD** [comment]
Thanks @srid! This also solved it for me, i.e. #60

**Pull request merged**